openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -41,6 +42,7 @@
 from util.date_utils import from_timestamp
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory
+import six
```

Comment:
It looks like python-modernize is just adding this to the end of the imports section, while our coding standard would put it higher (just below `import pytz` in this case).  We don't have `isort` running so this doesn't break any linting checks, but it might be nice to tidy this up.